src/Kopernicus/Components/KopernicusSolarPanels.cs

```diff
@@ -37,7 +38,16 @@ namespace Kopernicus.Components
     /// </summary>
     public class KopernicusSolarPanels : PartModule
     {
-        [KSPField(guiActive = true, guiActiveEditor = false, guiName = "Tracking Body", isPersistant = true)]
+        //Strings for Localization
+        string SP_status_DirectSunlight = Localizer.Format("#Kopernicus_UI_DirectSunlight");//"Direct Sunlight"
```

Comment:
I think the new unity might have some issues with this way of defining the variables.

you should change them into static strings

I am not sure if changing them to static is enough to fix all issues or if we should also define them at a different time

Reply:
Argh, you beat me to it. No, Unity won't have issues (using the constructor in a MonoBehaviour is totally fine as long as you don't create it with `new MB();`), but it is unneccessary to store a copy of these strings for every instance of the solar panel module.

Please make them `private static`.

Reply:
@tinygrox just do what @stolld said and we can merge your PR